packages/sirius-web/backend/sirius-web-services/src/test/java/org/eclipse/sirius/web/services/documents/UploadDocumentEventHandlerTests.java

```diff
@@ -193,6 +198,44 @@ public Optional<Document> createDocument(String projectId, String name, String c
         return editingDomain;
     }
 
+    private EditingDomain uploadDocumentAndCheckReport(InputStream inputstream, String expectedChangeKind, Class<?> expectedPayload) {
```

Comment:
While improving the test coverage of the old architecture is nice, it is not included in the code coverage report check on purpose. It will be better to improve `DocumentControllerIntegrationTests` since `UploadDocumentEventHandlerTests` will not stay in the codebase for a long time.

Reply:
I don't know how to add a new `IUploadDocumentReportProvider` just for a test in `DocumentControllerIntegrationTests`